applications/FluidDynamicsApplication/tests/test_fluid_rve.py

```diff
@@ -0,0 +1,96 @@
+import KratosMultiphysics
+import KratosMultiphysics.FluidDynamicsApplication as FluidDynamicsApplication # Might be needed for testing
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+
+import KratosMultiphysics.kratos_utilities as kratos_utilities # Might be needed for testing
+
+from KratosMultiphysics.FluidDynamicsApplication.fluid_rve_analysis import FluidRVEAnalysis
+
+# Previous line will import the Fluid RVE analysis class
+
+class TestFluidRVETest(KratosUnittest.TestCase):
+    
```

Comment:
```suggestion
    
    def setUp(self):
        self.print_output = False

```
By setting this to `True` we can print the test output for debugging. Then we remove the output value from the `rve_settings` so in a "standard" (out of the unit testing context) case we add the output as we do normally in the `output_process_list`.